rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -884,7 +905,7 @@ public TaskIdsPage getJobTasksIdsByTag(
                 tasksName.add(ts.getId().getReadableName());
             }
 
-            return new TaskIdsPage(tasksName, page.getSize());
+            return new RestPage<String>(tasksName, page.getSize());
         } catch (PermissionException e) {
             throw new PermissionRestException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)